ee/tables/execparsers/key_value/key_value.go

```diff
@@ -0,0 +1,41 @@
+//go:build darwin || linux
+// +build darwin linux
```

Comment:
Same q here, why darwin/linux only? Seems like it could be useful/applicable for Windows too